content/training/Walk-through/pop_and_lab_data.Rmd

```diff
@@ -69,17 +69,21 @@ reporting_week <- "2017-W16"  # Set the reporting week
 The population estimates for regions and age groups are collected in order to calculate attack and mortality rates. **The template allows several ways to provide population estimates, but importantly, the data must be stored under specific variable names.** In your real-use case, the specific age groupings and region names in your data may differ from the examples, but the variable names must follow the expected standard.  
 
 You can do one of the following:  
-1) Enter the estimates directly **[need clarification here as this is not yet built]**  
+1) Import a spreadsheet file with the estimates in the correct format  
 2) Use the function gen_population() to derive the estimates from proportions  
-3) Import a spreadsheet file with the estimates in the correct format  
 
 ![](/images/AJS_Population_Counts.png)  
 
-For the AJS example, we can use an Excel spreadsheet containing population figures by region. Uncomment this line of code in the read_population_data chunk and modify it to import the Excel spreadsheet (link for spreadsheet download at top of this webpage). Note that the spreadsheet variables names have already been edited to match the expected names ("patient_origin" and "population").  
+**For this case study, we import an Excel spreadsheet containing population data by quartier (available for download in case study overview page).**  
```

Comment:
Is this quartier or quarter?

```suggestion
**For this case study, we import an Excel spreadsheet containing population data by quarter (available for download in case study overview page).**  
```

Reply:
I see that is quartier, so Imma close this